server/controllers/clubAccountControllers.js

```diff
@@ -11,15 +12,39 @@ module.exports = {
         // TODO
     },
     getProfile: function(req, res){
-        // TODO
+        clubProfile.find( {clubAccountId: req.params.id} )
+            .then(clubprofile => res.json(clubprofile))
+            .catch(err => res.status(422).json(err))
     },
     getById: function(req, res){
-        // TODO
+        clubAccount.findById( {_id: req.params.id} )
+            .then(clubaccount => res.json(clubaccount))
+            .catch(err => res.status(422).json(err))
     },
     getByFirebaseId: function(req, res){
-        // TODO
+        clubAccount.find( {firebaseId: req.params.firebaseId} )
+            .then(clubaccount => res.json(clubaccount))
+            .catch(err => res.status(422).json(err))
     },
     getAll: function(req, res){
-        // TODO
+        //query for pending and sort, query for nonpending and sort
+        var q1 = clubAccount.find( {verificationStatus: 'pending'} )
+            .sort( {creationDate: -1, lastUpdateDate: -1} );
+        var q2 = clubProfile.find({
+            verificationStatus: {$in: ['accepted', 'denied']},
+            lastUpdateDate:  {
+                //Updated within the last 14 days
+                $gte: (new Date(ISODate().getTime() - 1000 * 3600 * 24 * 14))
```

Comment:
since the constraint of 14 days is an arbitrary configuration, could you add the number 14 as a variable in the configs file (`server/configs/index.js`) and then import it here, so that it's more standardized?